cns/restserver/api.go

```diff
@@ -1415,3 +1419,107 @@ func (service *HTTPRestService) deleteHostNCApipaEndpoint(w http.ResponseWriter,
 	err = service.Listener.Encode(w, &response)
 	logger.Response(service.Name, response, response.Response.ReturnCode, ReturnCodeToString(response.Response.ReturnCode), err)
 }
+
+// SetNodeOrchestrator :- Set node orchestrator after registering with mDNC
+func (service *HTTPRestService) SetNodeOrchestrator(r *cns.SetOrchestratorTypeRequest) {
```

Comment:
Looks like this is called internally by CNS. Can you move this to internalapi.go? This file only captures HTTP APIs exposed to external clients/components

Reply:
moved